### DISCUSSION TITLE
AYUDA PARA PRACTICA

### DISCUSSION THREAD

**Discussion by AngelArielGOOD** [Launch-X-Latam/MisionFrontEnd#41] [Q&A]
![image](https://user-images.githubusercontent.com/114241543/192626293-c0901064-2e0b-4bc3-b1d5-fecd04ee6e75.png)

Alguien que me pueda explicar como se llena, ya estuve un rato leyendo 2 veces la info y no entiendo como se llena, porfavor.

**Comment by erykzon** [comment]
No se llena todo , solo la información que es importante para tu proyecto el resto la puede eliminar.
